Tasks/Gradle/CodeAnalysis/SonarQube/common.ts

```diff
@@ -6,13 +6,15 @@ import fs = require('fs');
 
 import {ToolRunner} from 'vsts-task-lib/toolrunner';
 
+import tl = require('vsts-task-lib/task');
```

Comment:
Those are related to the interaction with the build system, but I don;'t think it's worth having a separate object to model that. having them as public static is fine as long as they don't complicate testing. For example in a simple class named "TfsBuild"